components/sup/src/supervisor.rs

```diff
@@ -104,7 +104,7 @@ pub struct Supervisor {
     pub package: Arc<RwLock<PackageInstall>>,
     pub preamble: String,
     pub state: ProcessState,
-    pub state_entered: SteadyTime,
+    pub state_entered: Timespec,
```

Comment:
Yeah I stumbled upon chrono in the serde issues and decided to skip the dependency since I would have to do some custom deserialization anyway to represent the time of the state change as elapsed seconds.